core/src/templates/components/quote/quote.twig

```diff
@@ -1,8 +1,10 @@
-<div class="su-quote">
+<figure class="su-quote">
   <img class="su-quote__img" src="{{ src }}" alt="{{ alt }}">
-  <div class="su-quote__body">
-    <p class="su-quote__heading">{{ name }}</p>
+  <section class="su-quote__body">
+    <h3 class="su-quote__heading">{{ name }}</h3>
     <p class="su-quote__bio">{{ bio }}</p>
-    <p class="su-quote__quote">"{{ text }}"</p>
-  </div>
-</div>
+    <blockquote>
+      <p class="su-quote__quote">"{{ text }}"</p>
+    </blockquote>
+  </section>
+</figure>
```

Comment:
Okay @sherakama, @yvonnetangsu, @JBCSU,
Here's a proposed HTMP5 approach for the on the quote. Your thoughts?

Reply:
I don't think `figure` makes sense here.  

From: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure
> Usually a <figure> is an image, illustration, diagram, code snippet, etc., that is referenced in the main flow of a document, but that can be moved to another part of the document or to an appendix without affecting the main flow.

The quote is not an image, illustration, or diagram to me.

Reply:
Ok, I just read through your comment thread. I still have mixed feelings about the figure tag but there seems to be a strong case for it. Ok.